core/tests/behavior/write.rs

```diff
@@ -578,6 +579,25 @@ pub async fn test_read_with_special_chars(op: Operator) -> Result<()> {
     Ok(())
 }
 
+// Delete existing file should succeed.
+pub async fn test_abort_writer(op: Operator) -> Result<()> {
+    let path = uuid::Uuid::new_v4().to_string();
+    let (content, _) = gen_bytes();
+
+    let mut writer = op.writer(&path).await.unwrap();
+    if let Err(e) = writer.append(content).await {
+        assert_eq!(e.kind(), ErrorKind::Unsupported);
```

Comment:
We should skip the entire test if `append` or `abort` not supported.

Reply:
> We should skip the entire test if `append` or `abort` not supported.

True, I forgot this